source/opt/ssa_rewrite_pass.h

```diff
@@ -57,10 +57,6 @@ class SSARewriter {
           is_complete_(false),
           users_() {}
 
-    PhiCandidate(const PhiCandidate&) = delete;
-    PhiCandidate(PhiCandidate&&) = default;
```

Comment:
You can't have a default move constructor in vs2013, but you can write one if you think it is needed.  It could be useful because of the two std::vector members.  However, I don't think it will have a bug performance impact, so this is fine for now.